src/Document.tsx

```diff
@@ -19,7 +21,20 @@ interface DocProps extends RouteComponentProps {
 
 const Document = (props: DocProps) => {
   const { docId } = props
+  const [docContent, setDocContent] = useState<Object>();
+
+  // const doc: any = {};
```

Comment:
Do you think we could remove these comments before merging?

Reply:
Yeah for sure. Need to take a look if it's possible to set up some linting rules to catch this.